helm-chart/cvat.values.yaml

```diff
@@ -3,51 +3,6 @@ analytics:
 
 ingress:
   enabled: true
-  annotations:
-    traefik.ingress.kubernetes.io/router.entrypoints: web
-    kubernetes.io/ingress.class: traefik
```

Comment:
You removed these annotations from here, but you didn't re-add them anywhere else. Is the user supposed to set them manually when enabling Traefik?

Reply:
There is no need to add the first line because this is the default behavior
The second line is deprecated, ingressclass is now used instead

Reply:
> There is no need to add the first line because this is the default behavior

Okay.

> The second line is deprecated, ingressclass is now used instead

Indeed, but `ingressClass` defaults to blank. Shouldn't it default to `traefik` when the `traefik` setting is true?

Reply:
done